src/dashboard.tsx

```diff
@@ -518,6 +536,30 @@ namespace Private {
     return url;
   }
 
+  /**
+   * Return the json result of /individual-plots.json
+   */
+  export function getItems(
+    url: string,
+    settings: ServerConnection.ISettings
+  ): Promise<IDashboardItem[] | void> {
+    url = normalizeDashboardUrl(url, settings.baseUrl);
+
+    return ServerConnection.makeRequest(
```

Comment:
This looks good to me. I think it could be made a bit flatter with async/await, but that's just cosmetic.

Reply:
I copied the `testDaskDashboard` pretty closely. :)